test/models/event_test.exs

```diff
@@ -18,8 +18,8 @@ defmodule Handsup.EventTest do
                    |> Group.changeset(%{name: Company.name,
                                         name_eng: Internet.slug(nil, "-") })
                    |> Repo.insert
-    attrs = %{ name: Venue.name, group_id: group.id}
-    changeset = Event.changeset(%Event{}, user, attrs)
+    attrs = %{ name: Venue.name}
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.